### PR TITLE
Photos comments

### PR DESCRIPTION
**Pull request by j311y**
Photos can be commented on. Refactored into partials.

### CODE REVIEW

File: app/models/photo_comment.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class PhotoComment < ApplicationRecord
+  belongs_to :photo
```
Comment (dbugsy) [inline app/models/photo_comment.rb:2]: This has the exact behaviour of the regular comment, so it's repeated code. There should be a better solution that is more dry.

**Review by dbugsy — CHANGES_REQUESTED** [review]
I don't see the reason for a PhotoComment model, better to reuse the existing comment class.

File: app/models/photo_comment.rb (hunk 1 above)
Comment (FiddlersCode) [reply app/models/photo_comment.rb:2]: @ddosque we couldn't figure out how to reconcile database conflicts - e.g. comments have a relationship with posts and photocomments have a relationship with photos...we thought it was cleaner to have them clearly separated

Comment (FiddlersCode) [reply app/models/photo_comment.rb:2]: http://guides.rubyonrails.org/association_basics.html#polymorphic-associations

ah...

Comment (FiddlersCode) [reply app/models/photo_comment.rb:2]: https://stackoverflow.com/questions/22025288/comments-on-multiple-models

**Pull request closed without merging**